easybuild/easyconfigs/p/PETSc/PETSc-3.17.4-cpeGNU-22.08.eb

```diff
@@ -24,10 +24,10 @@ dependencies = [
     ('METIS', '5.1.0'),
     ('ParMETIS','4.0.3'),
     ('SCOTCH', '6.1.3'),
-    ('MUMPS', '5.5.1', '-metis'),
+    ('MUMPS', '5.5.1',),
     ('Hypre', '2.26.0'),
-    ('SuperLU_DIST', '7.2.0'),
-    ('STRUMPACK', '6.3.1'),
+    ('SuperLU_DIST', '7.2.0', '-CPU'),
```

Comment:
In this case, `SuperLU_DIST`'s easyconfig needs then the line `versionsuffix = '-CPU'`.